README.md

```diff
@@ -124,3 +124,11 @@ nnoremap <Leader>rp :PythonCopyReferencePytest<CR>
 Usage
 -----
 Just place the cursor anywhere in the line with your target class/method/function name. Once you pressed your mappings, the reference will be copied in the system clipboard. Then, you could paste it anywhere.
+
+Testing
```

Comment:
Do you think it's worth having a GitHub Actions file to run this?

Reply:
I have no experience yet with Vim's Automated Tests, but if you could submit a PR would gladly merge it.

Reply:
That's odd: when I went to look at the tests I notice they weren't in the PR, nor are they in my local history. Sorry about that. I've made another PR to add the missing tests (and fix a bug): https://github.com/ranelpadon/python-copy-reference.vim/pull/2

Reply:
I also had a quick go making a GitHub Action to run the tests but it got fiddly as the version of Vim in GitHub's `ubuntu-latest` image doesn't support the "+" register. Probably not worth it.

Reply:
Yep, I noticed that they were not included, and I just thought that you'll probably do a follow-up PR.